UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -1835,69 +1685,20 @@ public void ClearPreview()
 				vecHit.y = origin.y + (float) RelativeY; // + offsetY;
 				//Check point here
 
-				if (LeftFaceHit)
-				{
-					// float TestX = ((vecHit.x - 0.5f) - Mathf.Floor(vecHit.x));
-
-					// if (0.05f < Math.Abs(TestX))
-					// {
-					// Logger.Log("Offsetuntouchx = " + Offsetuntouchx + "\n" + "directionx = " + direction.x + "\n" +
-					// "Step = " + xSteps + "\n" + "vexinv = " + vexinvX + "\n" + "offset = " + offsetX +
-					// "\n" + "\n" + "Test =" + TestX + "\n" + "Relative =" + (RelativeX)
-					// + "\n" + "\n"
-					// + " direction.x = " +  direction.x + " calculationFloat " + calculationFloat
-					// + "\n" + " xSteps " + xSteps + " gridOffsetx " + gridOffsetx  +"  vexinvX " + vexinvX);
-
-
-					// }
-				}
-				else
-				{
-					// float Testy = ((vecHit.y - 0.5f) - Mathf.Floor(vecHit.y));
```

Comment:
this is debug stuff, useful

Reply:
leaving commented out code is a bad practice. If we ever need it again, we use git.

Reply:
but am too dumb to use git,  change it to a debug bool?

Reply:
Why not a preprocessor, c# supports it.  Like #ifdef TILE_DEBUG #endif